src/time_series_parser.jl

```diff
@@ -16,9 +23,10 @@ mutable struct TimeSeriesFileMetadata
     normalization_factor::Union{String, Float64}
     "Path to the time series data file"
     data_file::String
+    "Resolution of the data being parsed in milliseconds"
+    resolution::Dates.Period
```

Comment:
@claytonpbarrows Note that this imposes a new requirement on the timeseries_pointers files.  They have to define `resolution`.  Previously the parsing code calculated it automatically.